src/app/cube/outage-page/past-issue/past-issue.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="card" [ngClass]="{ 'first': order.first, 'last': order.last }">
+  <div class="name">
+    <i class="{{ icon }}"></i>
```

Comment:
aria-hidden=true. This is so that screen readers ignore the icon because this is purely decorative.